mappings/net/minecraft/level/biome/Biome.mapping

```diff
@@ -22,20 +24,25 @@ CLASS kd net/minecraft/level/biome/Biome
 	FIELD w precipitates Z
 	FIELD x biomes [Lkd;
 	METHOD a createBiomeArray ()V
+	METHOD a getBiome (DD)Lkd;
+		ARG 0 x
```

Comment:
im assuming these are chunk coords so maybe chunkX/chunkZ? up to you

Reply:
we don't know that for sure 🤔

Reply:
okay fair enough